extensions/index.md

```diff
@@ -0,0 +1,16 @@
+---
+group: extensions
+title: Introduction
+---
+
+Magento extensions allow you to add custom features, functionality, services, and integrations to your Magento Admin and storefront. Extension developers package and install custom code as extensions, modules, components, addons, etc. You can design and develop extensions or purchase and install extensions from the Magento Marketplace or third-party providers.
+
+The [Magento Marketplace](https://marketplace.magento.com/) is the global eCommerce resource for applications and services that expand Magento solutions with powerful new features and functionality. Marketplace reviews and verifies all submitted extensions, modules, addons, and plugins are stable, supported, with clear documentation and release notes.
+
+This section tracks and provides installation guides and release notes for Magento developed and released packages.
+
+## Additional information
+
+* [Run the Module Manager]({{ page.baseurl }}/comp-mgr/module-man/compman-checklist.html)
```

Comment:
@lorikrell, you can't use the `{{page.baseurl}}` jekyll variable in versionless directories. You must use absolute links.

Reply:
Fixed!